pori_python/ipr/main.py

```diff
@@ -108,13 +108,7 @@ def command_interface() -> None:
     parser.add_argument(
         "-o",
         "--output_json_path",
-        help="path to a JSON to output the report upload body",
-    )
-    parser.add_argument(
-        "-w",
-        "--always_write_output_json",
-        action="store_true",
-        help="Write to output_json_path on successful IPR uploads instead of just when the upload fails",
```

Comment:
@dustinbleile is this safe to remove as far as the gsc report is concerned?

Reply:
@sshugsc it looks like this is actively used by the gsc report and can't be removed - https://svn.bcgsc.ca/bitbucket/projects/PROD/repos/gsc_report/browse/gsc_report/create_report.py#2030-2031,2104,2108,2110 . what about providing a default output path?